File: Easy/Python/551. Student Attendance Record I.py
class Solution(object):
    def checkRecord(self, s):
        countAbsent = 0

        for i in range(len(s)):
            if(s[i] == 'A'):
                countAbsent += 1
            if(i < len(s) - 2) and (s[i:i + 3] == "LLL"):
                return False
        
        return countAbsent < 2

File: Easy/Java/551. Student Attendance Record I.java
class Solution {
    public boolean checkRecord(String s) {
        int countAbsent = 0;

        for(int i = 0; i < s.length(); i++){
            if(s.charAt(i) == 'A')
                countAbsent++;
            if(i < s.length() - 2 && s.substring(i, i + 3).equals("LLL"))
                return false;
        }

        return countAbsent < 2;
    }
}
